commit bbb42412da6a02705ba3e668e90840c2683e4e67
Author: Jia Tan <jiat0218@gmail.com>
Date:   Tue Sep 26 00:47:26 2023 +0800

    Build: Remove Gnulib dependency from tests.
    
    The tests do not use any Gnulib replacements so they do not need to link
    libgnu.a or have /lib in the include path.

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 0523191d..ebc33a7b 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -1,81 +1,76 @@
 ##
 ## Author: Lasse Collin
 ##
 ## This file has been put into the public domain.
 ## You can do whatever you want with this file.
 ##
 
 EXTRA_DIST = \
 	files \
 	ossfuzz \
 	tuktest.h \
 	tests.h \
 	test_files.sh \
 	test_compress.sh \
 	test_compress_prepared_bcj_sparc \
 	test_compress_prepared_bcj_x86 \
 	test_compress_generated_abc \
 	test_compress_generated_random \
 	test_compress_generated_text \
 	test_scripts.sh \
 	bcj_test.c \
 	compress_prepared_bcj_sparc \
 	compress_prepared_bcj_x86 \
 	xzgrep_expected_output
 
 AM_CPPFLAGS = \
 	-I$(top_srcdir)/src/common \
 	-I$(top_srcdir)/src/liblzma/api \
-	-I$(top_srcdir)/src/liblzma \
-	-I$(top_builddir)/lib
+	-I$(top_srcdir)/src/liblzma
 
 LDADD = $(top_builddir)/src/liblzma/liblzma.la
 
-if COND_GNULIB
-LDADD += $(top_builddir)/lib/libgnu.a
-endif
-
 LDADD += $(LTLIBINTL)
 
 check_PROGRAMS = \
 	create_compress_files \
 	test_check \
 	test_hardware \
 	test_stream_flags \
 	test_filter_flags \
 	test_filter_str \
 	test_block_header \
 	test_index \
 	test_index_hash \
 	test_bcj_exact_size \
 	test_memlimit \
 	test_lzip_decoder \
 	test_vli
 
 TESTS = \
 	test_check \
 	test_hardware \
 	test_stream_flags \
 	test_filter_flags \
 	test_filter_str \
 	test_block_header \
 	test_index \
 	test_index_hash \
 	test_bcj_exact_size \
 	test_memlimit \
 	test_lzip_decoder \
 	test_vli \
 	test_files.sh \
 	test_compress_prepared_bcj_sparc \
 	test_compress_prepared_bcj_x86 \
 	test_compress_generated_abc \
 	test_compress_generated_random \
 	test_compress_generated_text
 
 if COND_SCRIPTS
 TESTS += test_scripts.sh
 endif
 
 clean-local:
 	-rm -f compress_generated_* \
 		xzgrep_test_output xzgrep_test_1.xz xzgrep_test_2.xz